warmup.krun

```diff
@@ -140,21 +140,24 @@ SKIP = [
 N_EXECUTIONS = 10  # Number of fresh processes.
 
 if sys.platform.startswith("linux"):
+    # Assumes systemd is used for init.
     PRE_EXECUTION_CMDS = [
-        "sudo service cron stop",
-        "sudo service atd stop",
-        "sudo service postfix stop",
-        "sudo service networking stop",
+        # Stop network first. If an interface accepts a DHCP lease during one
+        # of the later commands, it can cause them to fail.
```

Comment:
I had to parse this comment a couple of times before I understood. Maybe replace "of the later commands, it can cause them to fail." with "of the later commands, it can cause those commands to fail."